docs/contributors/manual-installation.md

```diff
@@ -41,117 +43,128 @@ active replica of `antrea-controller`.
 
 1. Grant the `antrea-controller` ServiceAccount necessary permissions to Kubernetes APIs. You can apply
 [controller-rbac.yaml](/build/yamls/base/controller-rbac.yml) to do it.
-```shell script
-kubectl apply -f build/yamls/base/controller-rbac.yml
-```
+
+    ```bash
+    kubectl apply -f build/yamls/base/controller-rbac.yml
+    ```
 
 2. Create the kubeconfig file that contains the K8s APIServer endpoint and the token of ServiceAccount created in the
 above step. See [Configure Access to Multiple Clusters](
 https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) for more information.
-```shell script
-APISERVER=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
-TOKEN=$(kubectl get secrets -n kube-system -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='antrea-controller')].data.token}"|base64 --decode)
-kubectl config --kubeconfig=antrea-controller.kubeconfig set-cluster kubernetes --server=$APISERVER --insecure-skip-tls-verify
-kubectl config --kubeconfig=antrea-controller.kubeconfig set-credentials antrea-controller --token=$TOKEN
-kubectl config --kubeconfig=antrea-controller.kubeconfig set-context antrea-controller@kubernetes --cluster=kubernetes --user=antrea-controller
-kubectl config --kubeconfig=antrea-controller.kubeconfig use-context antrea-controller@kubernetes
-```
+
+    ```bash
+    APISERVER=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+    TOKEN=$(kubectl get secrets -n kube-system -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='antrea-controller')].data.token}"|base64 --decode)
+    kubectl config --kubeconfig=antrea-controller.kubeconfig set-cluster kubernetes --server=$APISERVER --insecure-skip-tls-verify
+    kubectl config --kubeconfig=antrea-controller.kubeconfig set-credentials antrea-controller --token=$TOKEN
+    kubectl config --kubeconfig=antrea-controller.kubeconfig set-context antrea-controller@kubernetes --cluster=kubernetes --user=antrea-controller
+    kubectl config --kubeconfig=antrea-controller.kubeconfig use-context antrea-controller@kubernetes
+    ```
 
 3. Create the `antrea-controller` config file, see [Configuration](/docs/configuration.md) for details.
```

Comment:
Out of curiosity, this PR changes some absolute links to relative links, does it apply to sibling links only? what about these links that linked to a parent directory?

Reply:
You're right I forgot parent links, and those are broken on the website. I pushed a separate commit to fix those, and I also added a couple of CI checks to 1) validate that links used in the website are relative, and 2) check for broken links (using a third-party Github action, I found a couple links that were dead), so that I don't have to keep pushing PRs to fix website formatting issues (well at least not as often...). Everything is in a separate commit, so you just need to review that one.